### ISSUE TITLE
weather@mockturtl: Applet Bug

### ISSUE THREAD

**Issue by vlado-77** [linuxmint/cinnamon-spices-applets#6148]
### Applet version/Build date

Wetter (weather@mockturtl)

### Cinnamon version

20.3

### Distribution

weather@mockturtl out of order, what's up with it?

### Applet name and maintainer

weather@mockturtl @Gr3q

### What happened?

Keine Funktion! Wetter (weather@mockturtl)

### Other information

out of funktion since a week! What's happend?

**Comment by rcalixte** [comment]
> out of funktion since a week! What's happend?

Can you describe what is wrong with more detail?

**Comment by vlado-77** [comment]
Keine Funktion! Empfängt keine Wetterdaten, egal welcher freie Wetterdienst gewählt wird, findet keinen Standort wie Berlin oder Hamburg, Temperaturanzeige konstant  immer auf 25 Grad . Zig-mal  deinstalliert und neuinstalliert, ohne Erfolg.

**Comment by rcalixte** [comment]
> Keine Funktion! Empfängt keine Wetterdaten, egal welcher freie Wetterdienst gewählt wird, findet keinen Standort wie Berlin oder Hamburg, Temperaturanzeige konstant immer auf 25 Grad . Zig-mal deinstalliert und neuinstalliert, ohne Erfolg.

No function! Does not receive any weather data, no matter which free weather service is chosen, cannot find a location like Berlin or Hamburg, the temperature display is always constantly at 25 degrees. Uninstalled and reinstalled countless times without success.

**Comment by vlado-77** [comment]
No function! Does not receive any weather data, no matter which free weather service is chosen, cannot find a location like Berlin or Hamburg, the temperature display is always constantly at 25 degrees. Uninstalled and reinstalled countless times without success.

**Comment by rcalixte** [comment]
@vlado-77 This bug report is better than a discussion topic. The maintainer can respond when they have time. Please be patient.

**Comment by Gr3q** [comment]
@vlado-77 could you provide logs via Looking Glass? Without them i won't know what the problem is.

**Comment by vlado-77** [comment]
Nein, kann ich nicht weil ich nicht weiß wie das geht. Wetter app Mockturtl für LINUX mint 20.3 funktioniert seit einer Woche nicht mehr. Zig-mal de- und neu installiert. App funktioniert nicht mehr. Alle 5 freie Wetterdatenanbieter haben nach und nach keine Anzeige mehr. Mockturtl sucht pausenlos nach Info's aber empfängt keine. Ist Mockturtl nicht mehr unterstützt für Linux 20.3?  Warum kann man sie dann noch installieren und warum gibt es keinen LINUX-Support in deutsch oder zumindest deutschsprachige Foren? KI's mehrer Anbieter haben zu dem Problem keine Lösung, die sind schlau und blöd zugleich.

**Comment by rcalixte** [comment]
> Nein, kann ich nicht weil ich nicht weiß wie das geht. Wetter app Mockturtl für LINUX mint 20.3 funktioniert seit einer Woche nicht mehr. Zig-mal de- und neu installiert. App funktioniert nicht mehr. Alle 5 freie Wetterdatenanbieter haben nach und nach keine Anzeige mehr. Mockturtl sucht pausenlos nach Info's aber empfängt keine. Ist Mockturtl nicht mehr unterstützt für Linux 20.3? Warum kann man sie dann noch installieren und warum gibt es keinen LINUX-Support in deutsch oder zumindest deutschsprachige Foren? KI's mehrer Anbieter haben zu dem Problem keine Lösung, die sind schlau und blöd zugleich.

Translated from German:
No, I can't because I don't know how to do it. Weather app Mockturtl for LINUX mint 20.3 hasn't worked for a week. Uninstalled and reinstalled it countless times. App no ​​longer works. All 5 free weather data providers are gradually no longer displaying anything. Mockturtl is constantly searching for information but is not receiving any. Is Mockturtl no longer supported for Linux 20.3? Why can you still install it and why is there no LINUX support in German or at least German-speaking forums? AIs from several providers have no solution to the problem, they are smart and stupid at the same time.

@vlado-77:
There are some German speakers on the Matrix channels. For developer support, there are only so many German-speaking developers.

**Comment by vlado-77** [comment]
Bestimmt. Aber die kann ich nicht finden weil der Weg dahin ausschließlich englisch ist!  Wetter app Mockturtl für LINUX mint 20.3 funktioniert seit einer Woche nicht mehr. Zig-mal de- und neu installiert. App funktioniert nicht mehr. Alle 5 freie Wetterdatenanbieter haben nach und nach keine Anzeige mehr. Mockturtl sucht pausenlos nach Info's aber empfängt keine. Ist Mockturtl nicht mehr unterstützt für Linux 20.3? Warum kann man sie dann noch installieren?

**Comment by Gr3q** [comment]
If you can't provide logs (which can be done by going to the Applet Settings->Help then clicking on the "Save to File" button) then this is all I can tell you to do to test:

* Does it not work with Automatic locations?
* Does it not work with Manual Locations? 
* Does it not work with only entering lat,long in the manual location field in this format? `{lat},{long}`
* In the Applet Settings->Help could you set the User Agent String Override to anything other than empty and see if it works?
